docker-compose/docker-whisk-controller.env

```diff
@@ -13,6 +13,9 @@ DB_PREFIX=local_
 DB_WHISK_ACTIONS=local_whisks
 DB_WHISK_AUTHS=local_subjects
 DB_WHISK_ACTIVATIONS=local_activations
+DB_WHISK_ACTIONS_DDOC=whisks
+DB_WHISK_ACTIVATIONS_DDOC=whisks
```

Comment:
You’ll want to change these to whisk.v2

Reply:
oh thanks @rabbah . I was one week behind...